.circleci/config.yml

```diff
@@ -28,7 +28,7 @@ jobs:
   # muffet (https://github.com/raviqqe/muffet) to perform the link check.
   website_link_check:
     docker:
-    - image: cimg/node:18.12.1
+    - image: cimg/node:18.12.0
```

Comment:
Why is this downgraded

Reply:
this is forked branch. my forked `master` branch is force-pushed sometimes for renovate cheking. so this is no meaning diff. After force-pushed, renovate PR diff contains remain dirty diff.